src/control-plane/backend/lambda/api/service/quicksight/sql-builder.ts

```diff
@@ -38,8 +38,8 @@ export interface FunnelSQLParameters {
   readonly firstEventExtraCondition?: EventAndCondition;
   readonly eventAndConditions: EventAndCondition[];
   readonly timeScopeType: ExploreTimeScopeType;
-  readonly timeStart?: string;
-  readonly timeEnd?: string;
+  readonly timeStart?: Date;
+  readonly timeEnd?: Date;
```

Comment:
Pass date as **string** from client side and convert it into the date from server side is the best approach.

**clientside:**
```
{
"time" : "2023-08-01"
}
```

**Server side:**
`const date = new Date(req.body.time)`

Reply:
Does client side code using this interface?

Reply:
fixed